staging/com.smartnews/manifest.tt

```diff
@@ -0,0 +1,37 @@
+class @com.smartnews
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  list query top_articles(in opt counter: Number #[prompt="How many news articles do you want?"],
+    			  out news_id: String,
+                          out title: String,
```

Comment:
Every parameter should have a `#_[canonical]` annotation with different ways to refer to that parameter in different parts of speech. See the guide at <https://wiki.almond.stanford.edu/en/thingpedia/guide/natural-language>, under "Canonical Forms".